mzLib/MassSpectrometry/MsDataFile.cs

```diff
@@ -38,7 +39,8 @@ public abstract class MsDataFile
         public SourceFile SourceFile { get; set; }
         public int NumSpectra => Scans.Length;
         public string FilePath { get; }
-
+        public int[] OneBasedScanNumbers { get; set; }
```

Comment:
I don't think we need these. they can just be returned. and they definitely should not be settable

Reply:
it's not being used anymore with the latest commit